packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -99,8 +99,13 @@ function HeaderToolbar( {
 		? wasNoContentSelected.current
 		: noContentSelected;
 
+	/* translators: accessibility text for the editor toolbar */
+	const toolbarAriaLabel = __( 'Document tools' );
+
 	return (
 		<View
+			testID={ toolbarAriaLabel }
+			accessibilityLabel={ toolbarAriaLabel }
```

Comment:
I added these to be able to find them for E2E tests, since `testID` is not supported on Android we also have to add `accessibilityLabel` so it'd be for internal usage only.

Reply:
> `testID` is not supported on Android we also have to add `accessibilityLabel`

Aha. That is very helpful context. Thank you for explaining that. That is unfortunate. 😞 Is it worth leaving an inline comment stating it is currently only used for testing purposes?

Reply:
> Is it worth leaving an inline comment stating it is currently only used for testing purposes?

I know we have other cases like these around the codebase, so I'm not sure if having several comments explaining why would be the best approach. I feel like if it's next to a `testID` it could be self-explanatory once you know it is needed for E2E tests.